docs/src/components/TailwindClassnames/index.tsx

```diff
@@ -52,6 +69,49 @@ const getCategory = (name: string) => {
   return "palette";
 };
 
+const transformColors = (colors: Record<string, string>) => {
+  return Object.entries(colors).reduce((acc, [category, value]) => {
+    if (typeof value === "object") {
+      Object.entries(value).forEach(([name, color]) => {
+        acc[`${category}-${name}`] = color;
+      });
+    } else {
+      acc[category] = value;
+    }
+
+    return acc;
+  }, {});
+};
+
+const CategoryDescriptions = ({ category }: { category: Category }) => {
+  return (
+    <>
+      {category === "Colors" && (
+        <Text type="info" weight="medium" margin={{ bottom: "12px" }}>
+          To be used with Tailwind classes that apply a color property (eg: `text-blue-dark`,
```

Comment:
Apparently the `` don't have any effect (too used to MDX 😅) can we maybe use `<code>` or something to have it formatted? Or maybe just _italic_? WDYT?

Reply:
do not think anything is needed here, simple string is enough